src/Ajaxray/PHPWatermark/Watermark.php

```diff
@@ -160,6 +161,18 @@ public function setFontSize(int $fontSize): self
         return $this;
     }
 
+    /**
+     * Font color. Should to be in rgb or rgba format 
+     *
+     * @param string $fontColor
+     * @return Watermark
+     * Added by shqawe@gmail.com
+     */
+    public function setFontColor($fontColor)
+    {
+        $this->options['fontcolor'] = $this->_prepareColorForWaterMark($fontColor);
```

Comment:
To make this function chainable, we should return `$this`.
```suggestion
        $this->options['fontcolor'] = $this->_prepareColorForWaterMark($fontColor);
        return $this;
```

Reply:
You are correct I'm sorry i miss that.